examples/trials/mnist-bohb/mnist.py

```diff
@@ -0,0 +1,250 @@
+"""A deep MNIST classifier using convolutional layers."""
```

Comment:
add MIT license?

Reply:
All the trial codes in our current examples don't have an MIT license?